src/main/resources/swagger/api-docs.yaml

```diff
@@ -263,6 +263,50 @@ paths:
           description: Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
+  '/notebooks/{googleProject}/{clusterName}/api/contents':
```

Comment:
So this endpoint comes from Jupyter. Previously we've only been adding Swagger definitions for our Jupyter _extension_ APIs, not Jupyter APIs themselves. Jupyter has a lot more APIs which we don't document: https://github.com/jupyter/notebook/blob/master/notebook/services/api/api.yaml

My intuition is that we shouldn't be duplicating Jupyter's swagger inside Leo's swagger. I wonder if there's a way to "include" a swagger file from a different swagger file?